Library/Formula/truncate.rb

```diff
@@ -0,0 +1,18 @@
+class Truncate < Formula
+  desc "truncates a file to a given size"
+  homepage "http://www.vanheusden.com/truncate"
+  url "https://github.com/flok99/truncate/archive/fc30a29dab789249f0c5f6863fdc5c8d4c9c3f69.zip"
```

Comment:
Hi,

I'm not a git guru so I hope I did the right thing.
I did git tag -l "0.8" then commit and push.

On Thu, Jan 14, 2016 at 5:48 AM, Dominyk Tiller <notifications@github.com>
wrote:

> In Library/Formula/truncate.rb
> <https://github.com/Homebrew/homebrew/pull/47883#discussion_r49687419>:
>
> > @@ -0,0 +1,18 @@
> > +class Truncate < Formula
> > +  desc "truncates a file to a given size"
> > +  homepage "http://www.vanheusden.com/truncate"
> > +  url "https://github.com/flok99/truncate/archive/fc30a29dab789249f0c5f6863fdc5c8d4c9c3f69.zip"
>
> We need a tagged version here, I'm afraid. Hopefully upstream is
> able/willing to tag one for you, maybe worth asking them?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/47883/files#r49687419>.
>



-- 
www.vanheusden.com  www.slimwinnen.nl | www.smartwinning.info